### DISCUSSION TITLE
channel runs twice

### DISCUSSION THREAD

**Discussion by francescopozzoni** [nextgenhealthcare/connect#4740] [Q&A]
Hi everybody. Using mirth 3.5.1 and having some trouble.

I've got a channel with a javascript source that should run at 11.00 PM every day. I found that yesterday its source has received two messages at 11:00 PM (they differ by milliseconds), how is it possible?

**Comment by pacmano1** [comment]
If you are sure it is impossible for the channel to return more than one message on the JS source then check if some other channel routed to it.  Or an engineer posted to it.   Lastly of course check the event log to see if it in fact polled 2x.

**Comment by rjkroll** [comment]
What is your javascript source actually doing?   Are the messages identical?   

If you look and actually see 2 messages with a 'Sent' status in your dashboard, then your source returned two messages somehow.

**Comment by tonygermano** [comment]
if possible, please share an export of your channel
